app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,38 @@
+class RestaurantsController < ApplicationController
+
+  def index
+    @restaurants = Restaurant.all
+  end
+
+  def show
+    @reataurant = Restaurant.find(params[:id])
+  end
+
+  def new
```

Comment:
You don't need the new, edit and product_params methods for the moment, it's better to delete them